### ISSUE TITLE
Retention on psql-app

### ISSUE THREAD

**Issue by numchucksoe** [OpenBMP/obmp-docker#19]
I have the retention in the docker-compose file for 8 weeks but it is only doing 4 weeks. I do not see the old cron job that cleans up and can not find anywhere the dB is being cleaned.  Please advise.

**Comment by TimEvens** [comment]
The cron jobs have moved to postgres and TSDB retention policies.  This is a bug that those values are still in the compose file. A doc will be created to detail how to adjust, see, and monitor retention settings.

**Comment by numchucksoe** [comment]
Thanks a bunch. I was going crazy looking for it.  Look forward to the update and keep up good work.

**Comment by numchucksoe** [comment]
Any update on this?

**Comment by rylyk** [comment]
@numchucksoe are you waiting on the updated docs to get past your issue? I can assist modifying the retention policies if needed

**Comment by numchucksoe** [comment]
> @numchucksoe are you waiting on the updated docs to get past your issue? I can assist modifying the retention policies if needed

Yes I am waiting on a way to increase the retention. Any help will be greatly appreciated.

**Comment by rylyk** [comment]
Sure, you can see how the retention is set in the DB schema: [timescaledb retention policy](https://github.com/OpenBMP/obmp-psql/blob/fec3ba5a66bbeaf982178c680fd2eec4b2db30ba/database/2_aggregations.sql#L28)

You can connect to your database using psql.

1. List current retention policies on hypertables
  `SELECT * FROM timescaledb_information.jobs where proc_name='policy_retention';`
2. Drop whichever ones you want using [remove_retention_policy()](https://docs.timescale.com/api/latest/data-retention/remove_retention_policy/)
3. You can then replace with a new policy with longer retention to prevent your db from filling up [add_retention_policy()](https://docs.timescale.com/api/latest/data-retention/add_retention_policy/)
